test/screenshot/README.md

```diff
@@ -329,29 +351,32 @@ For example:
 
 1.  Keep HTML files _small_ and _focused_. This makes it easier to review diffs.
 
-2.  Each page should target a single logical "variant" of a component. For example:
+2.  Each page should generally target a single logical "variant" of a component. For example:
     - One "block" CSS class (e.g., `mdc-button`)
     - One "modifier" CSS class (e.g., `mdc-button--dense`)
     - One Sass mixin (e.g., `mdc-button-ink-color`)
 
 3.  File structure:
 
         /test/screenshot
-            /mdc-foo
-                /classes
-                    - baseline.html
-                    - dense.html
-                    - ...
-                /mixins
-                    - custom.scss
-                    - fill-color.html
-                    - filled-accessible.html
-                    - ink-color.html
-                    - ...
+            /spec
+                /mdc-foo
+                    /classes
+                        - baseline.html
+                        - dense.html
+                        - ...
+                    /mixins
+                        - fill-color.html
+                        - filled-accessible.html
+                        - ink-color.html
+                        - ...
+                    /fixture.js
+                    /fixture.scss
 
 4.  Do not test _combinations_ of mixins and modifier classes unless:
-    1. It's necessary to prevent a regression; or
-    2. We explicitly support the combination, and the implementation is likely to have bugs (e.g., `mdc-button--dense` and `mdc-button-outline-width` both set `line-height`)
+    1.  It's necessary to prevent a regression; or
+    2.  We explicitly support the combination, and the implementation is likely to have bugs \
+        (e.g., `mdc-button--dense` and `mdc-button-outline-width` both set `line-height`)
```

Comment:
This particular example is no longer true, we should remove it or find a new one

Reply:
Good catch! I created #3300 to update this.